### PR TITLE
MultitouchDragStrategy guide

### PR DESCRIPTION
**Pull request by Piinks**
_Description of what this PR is changing or adding, and why:_

Migration guide for https://github.com/flutter/flutter/pull/136708
Fixes https://github.com/flutter/flutter/issues/139333

## Presubmit checklist

- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

File: src/release/breaking-changes/multi-touch-scrolling.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+### Copy and modify existing `ScrollBehavior`
+
+Code before migration:
+
+```dart
+final ScrollController controller = ScrollController();
+ListView.builder(
+  controller: controller,
+  itemBuilder: (BuildContext context, int index) {
+   return Text('Item $index');
```
Comment (xu-baolin) [inline src/release/breaking-changes/multi-touch-scrolling.md:128]: nit: indent two spaces.

File: src/release/breaking-changes/multi-touch-scrolling.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+### Setting a custom `ScrollBehavior` for a specific widget
+
+Code before migration:
+
+```dart
+final ScrollController controller = ScrollController();
+ListView.builder(
+  controller: controller,
+  itemBuilder: (BuildContext context, int index) {
+   return Text('Item $index');
```
[the remark this answers is not in the document]
Comment (sfshaza2) [reply src/release/breaking-changes/multi-touch-scrolling.md:92]: I know this PR isn't complete, but please add the final code after running it through dart format.

[the remark this answers is not in the document]
Comment (Piinks) [reply src/release/breaking-changes/multi-touch-scrolling.md:92]: Hey Shams, can you clarify what else is incomplete?

**Review by sfshaza2 — APPROVED** [review]
lgtm.  Thanks, @Piinks!

**Pull request merged**